protokt-codegen/src/main/resources/message.stg

```diff
@@ -31,13 +31,13 @@ private constructor(
     <messagePropertyOverrides()><\\>
     val <p.name>: <p.propertyType><\\>
     }; separator=",\n"><if (properties)>,<endif>
-    val unknown: Map\<Int, Unknown> = emptyMap()
+    val unknownFields: UnknownFieldSet = UnknownFieldSet.empty()
```

Comment:
This default value is unneeded in Kotlin, but it is needed for our hobbled support of JSON deserialization.